src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -1,6 +1,6 @@
 package mate.academy.service;
 
-public class AuthenticationService {
+public class AuthenticationService extends UserService {
```

Comment:
In this case, it is better to use composition instead of inheritance

Reply:
Thank you for advice and revise.